wca-guidelines.md

```diff
@@ -163,8 +163,9 @@ To be more informative, each Guideline is classified using one of the following
 - A1a3+) [REMINDER] The organization team and the WCA Delegate must be mindful that time limits influence the strategies of the competitors (e.g. rushing the first two attempts in hopes of meeting a cutoff in a cutoff round), and that changing time limits after the start of the round can disadvantage some competitors unfairly.
 - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
 - A2c1+) [REMINDER] In the past, score sheets have been used to cover puzzles while leaving some sides exposed. This is no longer permitted.
-- A2d1+) [ADDITION] If a missing scramble signature is discovered before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
-- A2d1++) [ADDITION] If a missing scramble signature is discovered after the beginning of the attempt, the result may be kept, at the discretion of the WCA Delegate. However, the WCA Delegate should not allow such incidents to become common.
+- A2d1+) [ADDITION] If the scramble signature is found to be missing before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
+- A2d1++) [ADDITION] An attempt must not be disqualified solely due to a missing scramble signature. If a scramble signature is missing after the beginning of an attempt, the judge must allow the attempt to continue (if it is still in progress) and the WCA Delegate should allow the result to be kept. However, the WCA Delegate may replace the attempt with an extra attempt if they have a reason (other than the missing signature) to suspect that the scramble was incorrect.
+- A2d1+++) [REMINDER] Although the WCA Delegate may allow several individual attempts with missing scramble signatures to count, they should ensure that missing scramble signatures do not become common.
```

Comment:
It sounds a bit strange to me to call "they" to the Delegate/s
```suggestion
- A2d1+++) [REMINDER] Although the WCA Delegate may allow several individual attempts with missing scramble signatures to count, the WCA Delegate should ensure that missing scramble signatures do not become common.
```

Reply:
We do use "they" for the WCA Delegate in some places: https://www.worldcubeassociation.org/regulations/#4b2a

Repeating "WCA Delegate" sounds very redundant to my ears, so I'll stick with "they" if that's okay!
(I'm assuming it's okay with you because you already approved it.)